src/client/src/orangehrmTimePlugin/pages/timesheets/ViewEmployeeTimesheet.vue

```diff
@@ -41,8 +41,33 @@
         ></timesheet-period>
       </template>
       <template #footer-title>
-        <oxd-text v-show="timesheetStatus" type="subtitle-2">
-          {{ $t('general.status') }}: {{ timesheetStatus }}
+        <oxd-text
+          v-show="timesheetStatus"
```

Comment:
Please refactor this to a switch statement/ computed property. don't duplicate the template unnecessarily